pkg/client/cli/cmd_list.go

```diff
@@ -100,8 +118,17 @@ func (s *listInfo) list(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
+	if len(workloads) == 0 {
```

Comment:
Nit, but shouldn't this test be the very first thing that happens in the function?